projects/homework 6/sortedList/main.c

```diff
@@ -0,0 +1,53 @@
+#include "list.h"
+#include <stdio.h>
+
+int main()
+{
+    List* list = createList();
+    printf("\n0 – выйти\n1 – добавить значение в сортированный список");
+    printf("\n2 – удалить значение из списка\n3 – распечатать список\n");
+    while (1)
```

Comment:
```suggestion
    while (true)
```
Это то же самое, но более человекочитаемо